logger/Logger.h

```diff
@@ -46,18 +46,18 @@ namespace fair
 
 enum class Severity : int
 {
-    nolog,
-    fatal,
-    error,
-    warn,
-    state,
-    info,
-    debug,
-    debug1,
-    debug2,
-    debug3,
-    debug4,
-    trace,
+    nolog = 0,
+    fatal = 11,
+    error = 10,
+    warn = 9,
+    state = 8,
+    info = 7,
+    debug = 6,
+    debug1 = 5,
+    debug2 = 4,
+    debug3 = 3,
+    debug4 = 2,
+    trace = 1,
```

Comment:
While I like the changed order, only changing it here breaks a lot of checks in the implementation. Here is what I had to add to bring it back to life with your order: https://github.com/rbx/FairLogger/commit/a2e5a02cbb0bb467fd4d702d903193714195a825 (ignore the -NDEBUG, that's just for testing, although with this change we might want to add another build type).

Reply:
If we find a solution for my other comment, I can push these fixes to your branch.